### DISCUSSION TITLE
clarify roles of ; and | in expected values

### DISCUSSION THREAD

**Discussion by turbomam** [GenomicsStandardsConsortium/mixs#924] [General]
I thought 
- `|` meant select one from several options 
- `;` meant include the portion on both the left and right of the semicolon 

But maybe the usage is inconsistent?

There are a few examples in #345

**Comment by turbomam** [comment]
see also #329

**Comment by ramonawalls** [comment]
> I thought
> 
> * `|` meant select one from several options
> * `;` meant include the portion on both the left and right of the semicolon

That is how it is supposed to be. 
> 
> But maybe the usage is inconsistent?
> 
> There are a few examples in #345

Is this interfering with our ability to release MIxS6?

**Comment by turbomam** [comment]
> Is this interfering with our ability to release MIxS6?

No

**Comment by only1chunts** [comment]
I didn't see any actional issue here, so I've converted it to a discussion. If there is a actionable item that comes out of the discussion then we can open a ticket with a clear use case and criteria defined.
